attributes/default.rb

```diff
@@ -12,6 +12,11 @@
 default['onlinefs']['logs']                   = "#{node['onlinefs']['home']}/logs"
 default['onlinefs']['token']                  = "#{node['onlinefs']['etc']}/token"
 
+# Data volume directories
+default['onlinefs']['data_volume']['root_dir']  = "#{node['data']['dir']}/onlinefs"
+default['onlinefs']['data_volume']['etc_dir']   = "#{node['onlinefs']['data_volume']['root_dir']}/etc"
```

Comment:
`etc` is contains only configuration and a token that's regenerated every time. You don't need to move it to the `data_volume`

Reply:
As of now, the token is not regenerated when you restore a backup in hopsworks.ai